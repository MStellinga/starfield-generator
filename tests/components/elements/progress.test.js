/* eslint-disable jest/valid-expect */
import { h } from 'preact';
import Progress from '../../../src/components/elements/progress';
import render from 'preact-render-to-string';

describe('Test progress', () => {
	test('Progress renders correctly', () => {
        const tree = render(<Progress progress="50" />);
        expect(tree).toMatchSnapshot();
	});
});
