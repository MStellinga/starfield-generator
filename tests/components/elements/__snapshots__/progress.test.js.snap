// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test progress Progress renders correctly 1`] = `"<div class=\\"progress\\"><div class=\\"progressdone\\" style=\\"width: 50%;\\"></div></div>"`;
